src/main/resources/assets/wurst/lang/zh_cn.json

```diff
@@ -1,149 +1,170 @@
-  "description.wurst.hack.protect": "使用 AI 模式保护最近的实体，并保护它免受其他实体的伤害。\n使用指令 .protect 来保护指定的实体，而不是最近的一个。",
-  "description.wurst.hack.radar": "生成一个雷达 UI 并显示附近的实体。\n§c红色§r - 玩家\n§6橘色§r - 怪物\n§a绿色§r - 动物\n§7灰色§r - 其他",
-  "description.wurst.hack.rainbowui": "§c让§a每§9一§c样§a都§9充§c满§a五§9彩§c斑§a斓§9的§c样§a子§9，§c包§a括§9你§c当§a前§9所§c在 §aU§9I",
-  "description.wurst.hack.reach": "允许你碰到更远的距离。",
-  "description.wurst.hack.remoteview": "允许你以另一个视角观看世界。\n使用 .rv 指令来指定一个特殊的实体。",
-  "description.wurst.hack.safewalk": "防止你从方块边缘掉出。",
-  "description.wurst.hack.scaffoldwalk": "自动在你的脚下搭方块。",
-  "description.wurst.hack.search": "帮助你找到所需要的方块并以彩虹色的方式高亮。",
-  "description.wurst.hack.servercrasher": "生成一个物品可以崩掉 1.15.x 的服务器。\n§o需要创造模式。§r",
-  "description.wurst.hack.skinderp": "随机切换你的皮肤。",
-  "description.wurst.hack.sneak": "让你自动保持潜行行走。",
-  "description.wurst.hack.snowshoe": "可以让你在细雪上行走。",
-  "description.wurst.hack.speedhack": "允许你以大约 ~2.5x 或更快的速度跳跃或疾跑。\n\n§6§l警告：§r只有低于反作弊版本 3.13.2 才会工作且绕过，如果你不知道服务器反作弊是哪个版本。\n输入 §l/ncp version§r 来查看反作弊版本。",
-  "description.wurst.hack.speednuker": "一个比 Nuker 更快的版本，但会无法绕过反作弊。",
-  "description.wurst.hack.spider": "像蜘蛛一样能够爬墙。",
-  "description.wurst.hack.step": "允许你走上更高高度（取决于你的设定）的格子方块。",
-  "description.wurst.hack.throw": "迅速丢你手中种类的物品，可以用于丢雪球、鸡蛋、刷怪蛋、矿车，诸如此类。\n\n会生成大量的数量，这会产生大量的 lag 且可能会导致服务器崩溃。",
-  "description.wurst.hack.tillaura": "自动将泥土、草块、诸如此类。转化为耕地。\n不要和 Killaura 搞混。",
-  "description.wurst.hack.timer": "改变大部分东西的速度。",
-  "description.wurst.hack.tired": "让你看起来很像 2015 年 4 月的 Alexander。\n仅限其他玩家可视。",
-  "description.wurst.hack.toomanyhax": "屏蔽那些你并不想要有的功能。\n让你确保你不会不小心开启错误的作弊功能导致被服务器封禁。\n这个是给那些 “只想作弊一点点的用户”。\n\n使用指令 §6.toomanyhax§r 来选择哪些功能需要被屏蔽。\n输入 §6.help toomanyhax§r 寻求更多。",
-  "description.wurst.hack.tp-aura": "自动传送到实体附近并自动攻击他。",
-  "description.wurst.hack.trajectories": "预测箭和投掷物品的飞行路径。",
-  "description.wurst.hack.treebot": "一个实验性的机器人将会在树木附近砍树。\n目前仅限于砍小树。",
-  "description.wurst.hack.triggerbot": "自动攻击你所看的实体。",
-  "description.wurst.hack.trollpotion": "生成一个惹人恼怒的药水效果。\n§o需要创造模式。§r",
-  "description.wurst.hack.truesight": "允许你看到隐身的实体。",
-  "description.wurst.hack.tunneller": "自动挖一个隧道。\n\n§c§l警告：§r 尽管这个 AI 会自动避开岩浆和其他危险，无法确保你不会死亡，除非你不介意你有一定概率失去你的全部物品。",
-  "description.wurst.hack.x-ray": "允许你透视矿物所在的位置（请自行确保服务器内无假矿插件）。",
-  "description.wurst.altmanager.premium": "这个账户拥有密码，且可以加入所有服务器。",
-  "description.wurst.altmanager.cracked": "这个账户没有密码，只能加入盗版服务器。",
-  "description.wurst.altmanager.failed": "上次你尝试登录这个，但是它不起作用。",
-  "description.wurst.altmanager.checked": "这个密码在过去是有效的。",
-  "description.wurst.altmanager.unchecked": "你从未成功登录过这一个账户。",
-  "description.wurst.altmanager.favorite": "你标记了这其中一个账户为你的喜爱。",
-  "description.wurst.altmanager.window": "这个按钮是打开另一个窗口的。",
-  "description.wurst.altmanager.window_freeze": "当打开另一个窗口时，这会导致游戏看起来未响应。",
-  "description.wurst.altmanager.fullscreen": "§c请关闭全屏模式！"
+	"description.wurst.hack.anchoraura": "自动放置（可选）、充能、引爆重生锚并击杀你附近的实体。",

```

Comment:
It seems the latest commit has replaced all spaces with tabs. Please revert it to spaces.

Reply:
I have reverted the indent.